src/main/java/org/ibp/api/java/impl/middleware/dataset/validator/ObservationsTableValidator.java

```diff
@@ -96,4 +100,11 @@ private boolean validateIfValueIsMissingOrNumber(final String value) {
 		return NumberUtils.isNumber(value);
 	}
 
+	private boolean validateCategoricalVariableHasAPossibleValue(final MeasurementVariable var) {
+		if (var.getDataTypeId() !=null && var.getDataTypeId() == TermId.CATEGORICAL_VARIABLE.getId()) {
+			return var.getPossibleValues() != null;
```

Comment:
Hi @gelli-leafnode , is there a scenario where getPossibleValues returns an empty list instead of null?

Reply:
Hi Aldrin, 

I think the list of possible values will always be null if the field is empty. But I agree, condition to check if it's empty should be included. 

Thanks,
Gelli